File: old/Stones on the Table/1.py
def all_equal(array):
	return all(x==array[0] for x in array)

x = int(raw_input())
instone = raw_input()
n = len(instone)
count = 0
i=0
while i<x-1:
	if instone[i]==instone[i+1]:
		count+=1
	i+=1
if all_equal(instone):
	count = x-1
print count

File: old/Stones on the Table/1.rb
def all_equal(array)
	return array.max == array.min
end
x = gets.to_i
instone = gets.chomp().split("")
count =0
i = 0 
while i<x
	if instone[i]==instone[i+1]
		count+=1
	end
	i+=1
end
if all_equal(instone)
	count = x-1
end
puts count
